utils/fish/monero-wallet-cli.fish

```diff
@@ -0,0 +1,61 @@
+complete -c monero-wallet-cli -f
```

Comment:
```suggestion
complete -c monero-wallet-cli -f
```

`monero-wallet-rpc` and `monerod` don't have this line, can you explain it?

Reply:
this means that file completions for the flags are disabled by default and, if a flag accepts a file path as an argument, we then need to use `-F` in its definition. I will put this line in the other files for consistency

Reply:
Actually, it is also important for the file completionst that find files with a specific suffix. SO it is necessary, will add it